Sources/Nitrozen-SwiftUI/Elements/Element/NitrozenElementStyle.swift

```diff
@@ -29,6 +29,11 @@ public protocol NitrozenElementStringSelectableStyle {
 	var selectionTitle: String { get }
 }
 
+public protocol NitrozenElementRadioStringSelectableStyle {
+	var selectionTitleText: String { get }
+	var selectionSubTitleText: String? { get }
```

Comment:
Add todo comment with your name 
//TODO: Rushang -- combine NitrozenElementStringSelectableStyle and NitrozenElementRadioStringSelectableStyle with better naming

-> I will do these changes -- just add comment with your name and push